.github/workflows/deploy.yml

```diff
@@ -2,87 +2,86 @@ name: Deploy and Test
 
 on:
   push:
-    branches: ['master']
+    branches: ["master"]
     paths:
-    - helm-chart/**
-    - scripts/**
-    - acceptance-tests/**
-    - .github/**
+      - helm-chart/**
```

Comment:
vscode - at least for me - has some strong ideas about how YAML should be formatted. Do we actually have some rules defined for this?